internal/kube/adaptor/config_init.go

```diff
@@ -0,0 +1,47 @@
+package adaptor
+
+import (
+	"context"
+	"os"
+	paths "path"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/skupperproject/skupper/pkg/qdr"
+)
+
+func InitialiseConfig(client kubernetes.Interface, namespace string, path string, routerConfigMap string) error {
+	ctxt := context.Background()
```

Comment:
The config-init container logs are basically empty.
Maybe adding some basic info into it, just to see sslProfile secrets being syncronized, would be nice.